solutions/cron/README.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Vercel Cron Job Example
+slug: vercel-cron
+description: A Next.js app that uses Vercel Cron Jobs to update data at different intervals.
+framework: Next.js
+useCase:
+  - Cron
```

Comment:
Has this use case been created in Contentful already?

Reply:
Yessir!! https://vercel.com/templates/cron
![CleanShot 2023-02-21 at 17 56 59](https://user-images.githubusercontent.com/28986134/220485710-6b5b8005-e135-45a9-a68d-cc97e440981c.png)